databases/week1/homewrk.sql

```diff
@@ -0,0 +1,82 @@
+Use hyf_lesson1;
+/* Find out how many tasks are in the task table */
+select
+  COUNT(id)
+from
+  task;
+  /* Find out how many tasks in the task table do not have a valid due date */
+select
+  COUNT(id)
+from
+  task
+where
+  IsNull(due_date);
+  /* Find all the tasks that are marked as done */
+select
+  *
+from
+  task
+where
+  status_id = 3;
+  /* Find all the tasks that are not marked as done */
+select
```

Comment:
The same as I mentioned before, imagine you don't know what value 3 means. 
Try to think about how you can join both tables.